src/styles/user/user.sass

```
@@ -0,0 +1,42 @@
+$background-blue: #D9E1E9
+$primary-blue:  #0D4D7B
+$ligth-grey: #6e6c6cb0
+
+.wrapper
+  display: flex
+
+.primaryButton
+  background: $primary-blue
+.secondaryButton
+  background: $ligth-grey
```

```suggestion
.primaryButton
  background: $primary-blue
  border: none
.secondaryButton
  background: $ligth-grey
  border: none
```

Poistaa ainakin omasta mielestä ärsyttävän näköisen sinisen reunuksen sinisestä napista. Lähempänä imo myös figmaa näin.